src/components/set-sounds-invisible.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Sets all non-positional audio children to not be visible so their world matrices are not updated.
+ */
+AFRAME.registerComponent("set-sounds-invisible", {
```

Comment:
i suspect we could run into trouble here because mainline three.js doesn't have the constraint that "non-visible objects don't have their matrices updated" -- so a sane patch to A-Frame would be "set visible to false in the sound component" which would break positional audio for us